client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/IndicesClientDocumentationIT.java

```diff
@@ -784,6 +786,89 @@ public void onFailure(Exception e) {
         }
     }
 
+    public void testSyncedFlushIndex() throws Exception {
+        RestHighLevelClient client = highLevelClient();
+
+        {
```

Comment:
Can you remove the extra block? I don't think it buys anything.

Reply:
I mean, we still need to create the index, but i don't think we need to do it inside a new block.

Reply:
Yes but I think the reason here was to follow how it was done before (for eg: in `syncedFlush`). It kind of states that this is a pre-step. Before you test flushing an index it actually needs to exist. If you still think its unnecessary I will remove it.